composer.json

```diff
@@ -19,6 +19,7 @@
     }
   ],
   "require": {
```

Comment:
Pimcore 10 already enforces PHP8, no need for us to do it here too

Reply:
See comment made by @jorisros - also helps IDE to detect the correct PHP version to use.
Can remove if this is undesired.

Reply:
I guess it doesn't hurt, so we can leave it.